test-data/unit/check-expressions.test

```diff
@@ -323,7 +323,8 @@ if int():
     b = b or a  # E: Incompatible types in assignment (expression has type "Union[bool, A]", variable has type "bool")
 if int():
     b = a or b  # E: Incompatible types in assignment (expression has type "Union[A, bool]", variable has type "bool")
-class A: pass
+class A:
```

Comment:
Do we need these changes if the error code is disabled by default?

Reply:
(Would you rather me not do this right now?)

Reply:
Yes, I'd prefer not making changes not required. We'll make the changes when / if the error code is ever enabled by default.